typescript/infra/src/config/warp.ts

```diff
@@ -19,6 +19,9 @@ export const tokens: ChainMap<Record<string, Address>> = {
   base: {
     cbBTC: '0xcbB7C0000aB88B473b1f5aFd9ef808440eed33Bf',
     USDC: '0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913',
+    jc: '0x93aC8f81e799c3b76112A649715DD86CEa655BDF',
```

Comment:
This is a bigger issue beyond this PR, so feel free to resolve this, but wondering if we should use the token list standard for this